zio-http/shared/src/test/scala-3/zio/http/endpoint/OpenAPIGenSpec.scala

```diff
@@ -0,0 +1,87 @@
+package zio.http.endpoint.openapi
+
+import zio.test._
+
+import zio.schema.{Schema, derived}
+
+import zio.schema.DeriveSchema
+import zio.test._
+import zio.http.endpoint.Endpoint
+import zio.http.codec.{Doc, HttpCodec}
+import zio.http.endpoint.Endpoint
+import zio.http.{MediaType, RoutePattern}
+import zio.schema.annotation.{caseName, discriminatorName}
+import zio.prelude.Subtype
+
+object OpenAPIGenSpec extends ZIOSpecDefault {
```

Comment:
I'm not sure where the error comes from, so I copy/pasted a simplified version of my endpoint generating the error to reproduce it in the tests